compose.yml

```diff
@@ -32,13 +31,13 @@ services:
             - MW_RECAPTCHA_SITE_KEY
             - MW_RECAPTCHA_SECRET_KEY
             - MW_ADMIN_USER
-            - MW_ADMIN_PASS
+            - MW_ADMIN_PASS_FILE=/run/secrets/mw_admin_pass
             - MW_DB_NAME
             - MW_DB_INSTALLDB_USER=root
-            - MW_DB_INSTALLDB_PASS
+            - MW_DB_INSTALLDB_PASS_FILE=/run/secrets/db_root_password
             - MW_DB_USER=root
-            - MW_DB_PASS=$MW_DB_INSTALLDB_PASS
-            - MW_SECRET_KEY
+            - MW_DB_PASS_FILE=/run/secrets/db_root_password
```

Comment:
Same here

Reply:
yes, but this will work too

Reply:
Let's document this somewhere + related https://wikiteq.atlassian.net/browse/WIK-1735